confluent_kafka/src/Consumer.c

```diff
@@ -55,7 +55,7 @@ static void Consumer_dealloc (Handle *self) {
 	if (self->rk)
 		rd_kafka_destroy(self->rk);
 
-	Py_TYPE(self)->tp_free((PyObject *)self);
+        Py_TYPE(self)->tp_free((PyObject *)self);
```

Comment:
The file originally has tabs for indention, which sucks but I probably coded it with the wrong .emacs file in place.
New changes use space for indent instead.
We can do a forklift untabify at some point.